plugins/woocommerce/src/Blocks/Domain/Services/CheckoutFieldsFrontend.php

```diff
@@ -186,41 +187,41 @@ public function edit_account_form_fields() {
 	 * @param integer $user_id User ID.
 	 */
 	public function save_account_form_fields( $user_id ) {
-		// phpcs:disable WordPress.Security.NonceVerification.Missing
 		$customer          = new WC_Customer( $user_id );
-		$additional_fields = $this->checkout_fields_controller->get_fields_for_location( 'contact' );
-		$field_values      = array();
-
-		foreach ( array_keys( $additional_fields ) as $key ) {
-			$post_key = CheckoutFields::get_group_key( 'other' ) . $key;
-			if ( ! isset( $_POST[ $post_key ] ) ) {
-				continue;
-			}
+		$additional_fields = $this->sanitize_additional_fields_for_location_group( $customer, 'contact', 'other' );
+
+		// Generate document object based on POSTed values.
+		$document_object = null;
+
+		if ( Features::is_enabled( 'experimental-blocks' ) ) {
+			$document_object = new DocumentObject(
+				[
+					'customer' => [
+						'additional_fields' => $additional_fields,
+					],
```

Comment:
This is the contact fields? did we do that as well on Checkout?

Reply:
Yes checkout uses this key for contact fields too. https://github.com/woocommerce/woocommerce/blob/0096bc5021df4e63862d709e66c40b22b622ebaf/plugins/woocommerce/src/StoreApi/Routes/V1/Checkout.php#L192